### PR TITLE
[DTMF Dialer] Update Dialpad page in storybook

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Update the Dialpad component page in storybook
# Why

Highlights the new modes for the dialpad and talks a little about scenarios that the new modes are typically used for

https://skype.visualstudio.com/SPOOL/_workitems/edit/3564639
# How Tested

Locally in storybook, check out the storybook link when ready

### CODE REVIEW

**Comment by dmceachernmsft** [comment]
Passed the storybook link to Tay, will wait for his feedback on the content before merging 😊

**Review by JamesBurnside — APPROVED** [review]

File: packages/storybook/stories/Dialpad/Dialpad.stories.tsx
Comment (edwardlee-msft) [inline packages/storybook/stories/Dialpad/Dialpad.stories.tsx:30]:
```suggestion
        The Dialpad component is versatile in its usage. Like on a physical phone, the dialpad can have many different
        roles. Our Dialpad component is designed to accommodate different scenarios encountered in calling
        experiences.
```

File: packages/storybook/stories/Dialpad/Dialpad.stories.tsx
Comment (edwardlee-msft) [inline packages/storybook/stories/Dialpad/Dialpad.stories.tsx:36]:
```suggestion
        This is the default mode of the Dialpad component. In this mode you are able to enter and edit numbers in the
        Dialpad's input box. The typical scenario to use this mode is when you are dialling to phone number to call the number, or to dial
        in a new participant into an ongoing call.
```

File: packages/storybook/stories/Dialpad/Dialpad.stories.tsx
Comment (edwardlee-msft) [inline packages/storybook/stories/Dialpad/Dialpad.stories.tsx:46]:
```suggestion
        This mode is for sending DTMF tones when in a call. These tones are used for controlling bots or other services
        that you might encounter. Each tone is mapped to a sound that is played when pressing each key.
        You can disable these sounds by using the `disableDtmfPlayback` property on the component. In this mode, the
        input box in the dialpad is hidden since you are not able to edit DTMF tones that are sent.
```

File: packages/storybook/stories/Dialpad/Dialpad.stories.tsx
Comment (edwardlee-msft) [inline packages/storybook/stories/Dialpad/Dialpad.stories.tsx:60]:
```suggestion
        We provide many different ways to customize the Dialpad component. This example showcases how to customize the
        format for dialpad input using onChange, how to grab textfield values using onChange, and how to add extra
        functionality to dialpad buttons. This overrides the included number formatting behavior to North
        American phone numbers. In this example, the sounds are also disabled.
```

File: packages/storybook/stories/Dialpad/Dialpad.stories.tsx
Comment (edwardlee-msft) [inline packages/storybook/stories/Dialpad/Dialpad.stories.tsx:35]:
```suggestion
        Dialpad's input box. The typical scenario to use this mode is when you are dialling a phone number to call the number, or to dial
```

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**
